docs/modules/commons/pages/table-transformers.adoc

```diff
@@ -516,3 +516,77 @@ Examples:
 |value |1    |
 |value2|0    |
 ----
+
+== INNER_JOIN
+
+`INNER_JOIN` transformer combines rows from two tables whenever there are matching values between the columns.
+
+[cols="1,3", options="header"]
+|===
+|Parameter
+|Description
+
+|`leftTableJoinColumn`
+|the name of the column in the first table for matching
+
+|`rightTableJoinColumn`
+|the name of the column in the second table for matching
+
+|`tables`
+|paths to examples tables or transformers to inner join
+|===
+
+[IMPORTANT]
+====
+* number of unique tables paths must be equal 2
+* tables must have unique headers(keys), but the column names for matching can be the same in both tables
+* if any of tables is empty - the resulting table will also be empty
+====
+
+.Usage of INNER_JOIN transformer with the same column names for matching
+[source,gherkin]
+----
+{transformer=INNER_JOIN, leftTableJoinColumn=joinID, rightTableJoinColumn=joinID, tables=story/tables/test1.table;story/tables/test2.table}
+----
+
+.Usage of INNER_JOIN transformer with the different column names for matching
+[source,gherkin]
+----
+{transformer=INNER_JOIN, leftTableJoinColumn=joinID1, rightTableJoinColumn=joinID2, tables=story/tables/test1.table;story/tables/test2.table}
+----
+
+.Usage of INNER_JOIN transformer with nested tables
+[source,gherkin]
+----
+{transformer=INNER_JOIN, leftTableJoinColumn=joinID, rightTableJoinColumn=joinID, tables=\{transformer=FROM_EXCEL\, path=TestTemplate.xlsx\, sheet=Data\, addresses=A2\;A3\, column=test1\};\{transformer=FROM_EXCEL\, path=TestTemplate.xlsx\, sheet=Data\, range=B2:B4\, column=test2\}}
+----
+
+.Usage of INNER_JOIN transformer with table body
+[source,gherkin]
+----
+{transformer=INNER_JOIN, leftTableJoinColumn=joinID1, rightTableJoinColumn=joinID2, tables=/data/some-table.table}
+|joinID|column2|
+|5     |row25  |
+|3     |row23  |
+|1     |row21  |
+----
```

Comment:
this example doesn't seem valid because we don't have joinID1 or joinID2 in second table

Reply:
fixed, thank you